src/ManageCourses.Domain/DatabaseAccess/IManageCoursesDbContext.cs

```diff
@@ -50,6 +50,8 @@ public interface IManageCoursesDbContext
         List<UcasCourse> GetUcasCourseRecordsByInstCode(string instCode, string email);
         IQueryable<McOrganisationInstitution> GetUserOrganisations(string email);
         McOrganisationInstitution GetUserOrganisation(string email, string instCode);
+
+        IQueryable<McUser> GetMcUsers(string email);
```

Comment:
Why not just use the dbset directly? I don't like all these extra methods duplicating and restricting what you can already do with the dbsets.

The dbset can be added to the interface if it's missing.

minor but strongly dislike this existing pattern

Reply:
coz it needs custom implementation, see implementation

Reply:
ah, lowercaseness